docs/validators.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Validation functions
+author: Roxane Letourneau
+---
+
+Taquito provides functions that allow us to see if an address, a chain, a key hash, a contract address, a public key or a signature is valid. The `ValidationResult` returned by these functions is an enum which can take the following values:
+  ```
+  0 = NO_PREFIX_MATCHED,
+  1 = INVALID_CHECKSUM,
+  2 = INVALID_LENGTH,
+  3 = VALID
+  ```
+
+### Validate an address
+
+This function can be used to validate addresses as well as contract addresses. Calling the `validateAddress` function with the public key hash (pkh) from a faucet (https://faucet.tzalpha.net/) will return valid :
+
+```js live noInline
+import { validateAddress } from '@taquito/utils';
+
+const pkh = 'tz1L9r8mWmRPndRhuvMCWESLGSVeFzQ9NAWx'
+const validation = validateAddress(pkh)
+println(validation)
+    
+```
+
+Calling the `validateAddress` function with the previous pkh missing its last character will result in an invalid checksum :
```

Comment:
Instead of making the address too short, better to make it invalid, as this will impress upon the reader that there's actually an integrity check. 

For example:
`tz1L9r8mWmRPndRhuvMCWESLGSVEFzQ9NAWx` <-- Invalid
`tz1L9r8mWmRPndRhuvMCWESLGSVeFzQ9NAWx` <-- Valid